packages/flutter/lib/src/rendering/animated_size.dart

```diff
@@ -0,0 +1,141 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/animation.dart';
+import 'package:meta/meta.dart';
+
+import 'box.dart';
+import 'object.dart';
+import 'shifted_box.dart';
+
+/// A render object that can render child size change animations. It works by
+/// always animated from its old child's size to the new one with the given
+/// duration and curve. In case the child changes its size in every single frame
+/// the object deduces that its size is animated from somehwere else, it
+/// skips/stops its animation and immediately changes its size to match the
+/// child.
+///
+/// In case the child overflows the current animated size of the parent, it gets
+/// clipped automatically.
+class RenderAnimatedSize extends RenderAligningShiftedBox {
```

Comment:
"It works by" -> "It"
"In case" -> "If" (twice)
Remove "deduces that its size is animated from somehwere else"

there's some grammar that will need fixing once you've made those changes